patch/llpcPatchResourceCollect.h

```diff
@@ -87,12 +95,91 @@ class PatchResourceCollect:
     std::unordered_set<uint32_t>    m_importedOutputLocs;       // Locations of imported generic outputs
     std::unordered_set<uint32_t>    m_importedOutputBuiltIns;   // IDs of imported built-in outputs
 
+    std::vector<llvm::CallInst*>    m_inOutCalls;               // The import or export calls
+
     bool            m_hasPushConstOp;           // Whether push constant is active
     bool            m_hasDynIndexedInput;       // Whether dynamic indices are used in generic input addressing (valid
                                                 // for tessellation shader, fragment shader with input interpolation)
     bool            m_hasDynIndexedOutput;      // Whether dynamic indices are used in generic output addressing (valid
                                                 // for tessellation control shader)
     ResourceUsage*  m_pResUsage;                // Pointer to shader resource usage
+    std::unique_ptr<InOutLocationMapManager> m_pLocationMapManager; // Pointer to InOutLocationMapManager instance
+};
+
+// =====================================================================================================================
```

Comment:
Remove this. Sepearte line is only for class and function body.

Reply:
Thanks. Fixed